test/cpp/api/modules.cpp

```diff
@@ -1060,6 +1061,22 @@ TEST_F(ModulesTest, MaxPool3d_MaxUnpool3d) {
   ASSERT_EQ(unpooled_output.sizes(), std::vector<int64_t>({20, 16, 51, 33, 15}));
 }
 
+TEST_F(ModulesTest, Bias) {
+  Bias model(2);
+  auto input = torch::randn({2}, torch::requires_grad());
+  auto output = model(input);
+  torch::Tensor s = output.sum();
+
+  s.backward();
+  ASSERT_THAT(output.sizes(), testing::ElementsAre(2));
```

Comment:
Alternatively to using gmock for this assert, you could do what the previous test does:
```
ASSERT_EQ(output.sizes(), std::vector<int64_t>({2}));
```

Reply:
Oh nice. My memory told me that this used to give a poor output, now it's much better.

```
  auto sizes = std::vector<int64_t>{3, 5, 7, 11, 13};
  EXPECT_EQ(sizes, (std::vector<int64_t>{5, 7}));
  EXPECT_THAT(sizes, testing::ElementsAre(5, 7));
```

`EXPECT_EQ`:
```
Expected equality of these values:
  sizes
    Which is: { 3, 5, 7, 11, 13 }
  (std::vector<int64_t>{5, 7})
    Which is: { 5, 7 }
```

`EXPECT_THAT`
```
Value of: sizes
Expected: has 2 elements where
element #0 is equal to 5,
element #1 is equal to 7
  Actual: { 3, 5, 7, 11, 13 }, which has 5 elements
```

Very comparable! Advantages to `ElementsAre`: you get counts and you have an explicit distinction between expected and actual. But that's not that compelling. I'll change for this PR to be consistent.